ampersand-array-input-view.js

```diff
@@ -183,8 +174,15 @@ module.exports = View.extend({
         this.fields = without(this.fields, field);
         field.remove();
         this.fieldsRendered -= 1;
+        this._setRemoveableFields();
         this.update();
     },
+    _setRemoveableFields: function () {
+        var removable = this.fields.length > (this.minLength || 1);
```

Comment:
I'm usually for keeping with previous behavior but not as much as I'd like to stop treating 0 as false in my code.

I'd +1 this with `this.fields.length > this.minLength`